collector/metrics.go

```diff
@@ -42,13 +38,19 @@ func New(hostname string, port int, login string, password string) *Exporter {
 
 		hostname,
 		port,
+
+		prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: namespace,
+			Name:      "up",
+			Help:      "Was the last query of squid successful?",
+		}, []string{"region"}),
```

Comment:
Not sure why you chose `region` as label for this metric, but I think `host` is more appropriate:
```suggestion
		}, []string{"host"}),
```

Reply:
Oh! Now I understood what's happening, the original label was `region` and I don't remember why  🤦‍♂️ . Any way I think the code needs to be updated in two other places further down as well.

Reply:
I changed the label name in the other places and should be fine now. But please be aware that this change may be a breaking change for other users of the exporter.

Reply:
I was talking about the label name could be a problem but since it was added a couple of days ago it shouldn't be that big of a deal.